components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -20,6 +20,27 @@ typedef void (^MDCFlexibleHeaderChangeContentInsetsBlock)(void);
 typedef void (^MDCFlexibleHeaderShadowIntensityChangeBlock)(CALayer *_Nonnull shadowLayer,
                                                             CGFloat intensity);
 
+/** This determines how we adjust the height to account for the Safe Area insets. */
+typedef NS_ENUM(NSInteger, MDCFlexibleHeaderSafeAreaAdjustsMinMaxHeight) {
```

Comment:
I'm wondering if we can make this API define what changes from the client's point of view, rather than what changes internally. Right now the API says "safe area adjusts min/max height" which is, to me, an implementation detail.

Do you feel like `MDCFlexibleHeaderHeight` and `.includesSafeArea`/`.doesNotIncludeSafeArea` more clearly communicates how the `minimumHeight`/`maximumHeight` values are to be set?